continuous_integration/install.sh

```diff
@@ -76,3 +76,4 @@ if [ -z "$NUMPY_VERSION" ]; then
 fi
 
 python setup.py install
+pip install nose-exclude
```

Comment:
> The reason I am saying that is that the doctests are written in a slightly
> peculiar ways: the imports are not done in each doctest so the reason it seem
> to work magically is because there is a global state somewhere.

I think that this is actually the standard way to right doctest :)